src/loggedIO/loggedIO.ts

```diff
@@ -205,9 +205,7 @@ namespace Playback {
         return log;
     }
 
-    function initWrapper(wrapper: PlaybackSystem, underlying: ts.System): void;
-    function initWrapper(wrapper: PlaybackIO, underlying: Harness.IO): void;
-    function initWrapper(wrapper: PlaybackSystem | PlaybackIO, underlying: ts.System | Harness.IO): void {
+    export function initWrapper(...[wrapper, underlying]: [PlaybackSystem, ts.System] | [PlaybackIO, Harness.IO]): void {

```

Comment:
I can revert this if anybody cares, it's an egregious overload replaced with an egregious variadic parameter